Refact cartoon journal record

ivanivlad opened the pull request:
Задача "Рефакторинг Cartoon и JournalRecord"
Рефакторинг JournalRecord потребовал изменений в UtilizationFactory

ivanivlad committed:
Рефакторинг картона

ivanivlad committed:
Рефакторинг журнала записей

ikarpov64 commented on src/main/java/com/javaacademy/Cartoon.java:
```diff
@@ -1,27 +1,19 @@
-    public void setWeight(double weight) {
-        this.weight = weight;
-    }
+    public double weight;
```
privat double weight;

ikarpov64 replied:
privat double weight;

ivanivlad committed:
Исправление замечаний с ревью
"В классе картон, поле масса тоже стоит паблик, наверное следует исправить на приват"

ivanivlad committed:
Merge branch 'main' into refact_Cartoon_JournalRecord

korelov commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Рефакторинг Cartoon и JournalRecord](https://trello.com/c/ujnfYaUy/31-%D1%80%D0%B5%D1%84%D0%B0%D0%BA%D1%82%D0%BE%D1%80%D0%B8%D0%BD%D0%B3-cartoon-%D0%B8-journalrecord)
